ex24/main.go

```diff
@@ -1,5 +1,29 @@
 package main
 
-  func main() {
+import (
+	"fmt"
+	"math"
+)
 
-  }
+type Point struct {
+	x, y int
+}
+
+//func (p *Point) Construct(x, y int) {
+//	p.x = x
+//	p.y = y
+//}
+
+func newPoint(x, y int) *Point {
```

Comment:
It's better to use `NewPoint` if we gonna export this struct

Reply:
What does it mean to export a structure?

Reply:
This task is about how you understand packages and scopes. If you place this struct inside a package, you should export the `Point` interface and make sure that you can create a `point` struct outside of the package. This is why you want to have something like this:
```go
func NewPoint(x, y int) Point {
    return &point{x,y}
}
```